src/Component/Counter.js

```diff
@@ -0,0 +1,19 @@
+
+import React from 'react';
+import Button from './Button';
+
+function Counter({ setFruitteller, fruitTeller }) {
+    return (
+        <>
+            <Button type="button" disabled={fruitTeller === 0} clickHandler={() => setFruitteller(fruitTeller - 1)}>
+                -
+            </Button>
+            <p>{fruitTeller}</p>
+            <Button type="button" clickHandler={() => setFruitteller(fruitTeller + 1)}>
```

Comment:
Nu zal telkens wanneer `fruitTeller` veranderd een onnodige rerender van deze functie plaatsvinden. Wat je zou kunnen doen is de huidige waarde gebruiken die je vanuit de setValue functie al krijgt:
```suggestion
            <Button type="button" clickHandler={() => setFruitteller(currentValue => currentValue + 1)}>
```